src/main/commandServer/httpCommandServer.ts

```diff
@@ -89,7 +116,7 @@ export class HttpCommandServer {
 
         return;
       }
-      await command.call(this, request, response);
+      await command.call(this, request, response, { interactive: user === 'interactive' });
```

Comment:
Huh? At line 38 the user is set to `'interactive-user'`. So is the interactive user behaviour not actually tested?